### DISCUSSION TITLE
Can I configure jaeger-query(frontend) to read traces from Tempo?

### DISCUSSION THREAD

**Discussion by manojm321** [grafana/tempo#2919] [Q&A] [answered]
Hello,

We are migrating away from jaeger-collector/Elasticsearch backend. But I'd like to keep using jaeger-query frontend to visualize traces

**Comment by mapno** [comment, marked as answer]
Hi. you can find an adaptor to query Tempo with jaeger-query in [`cmd/tempo-query`](https://github.com/grafana/tempo/tree/main/cmd/tempo-query). You'll be missing on a lot of Tempo features though!
